roles/ansible_service_broker/tasks/install.yml

```diff
@@ -17,10 +17,12 @@
     ansible_service_broker_etcd_image_etcd_path: "{{ ansible_service_broker_etcd_image_etcd_path | default(__ansible_service_broker_etcd_image_etcd_path) }}"
 
     ansible_service_broker_registry_type: "{{ ansible_service_broker_registry_type | default(__ansible_service_broker_registry_type) }}"
+    ansible_service_broker_registry_name: "{{ ansible_service_broker_registry_name | default(__ansible_service_broker_registry_name) }}"
```

Comment:
This can probably just be defaulted in `defaults/vars.yml`
```
ansible_service_broker_registry_name: Foo
```

Then just refer to variable here.

Reply:
the default value will change between origin and openshift-enterprise